clients/js/cmds/sui.ts

```diff
@@ -65,6 +65,12 @@ exports.builder = function(y: typeof yargs) {
       return yargs
         .option("network", network_options)
         .option("rpc", rpc_description)
+        .option("package-id", {
+          alias: "pid",
```

Comment:
maybe drop this alias, `pid` is confusing as it conventionally means process id

Reply:
also, won't discovering the deployer capability (from my other comment) tell us what the package id is?

Reply:
definitely have to find the one that corresponds to the `package-id` argument, but that should be simple